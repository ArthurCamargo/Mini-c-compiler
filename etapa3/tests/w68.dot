digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n2;
n2 -> n3;
n3;
n3 -> n4;
n3 -> n5;
n3 -> n6;
n4;
n4 [label="10"];
n5;
n5 [label="11"];
n6;
n6 -> n7;
n6 -> n8;
n6 -> n9;
n7;
n7 [label="20"];
n8;
n8 [label="21"];
n9;
n9 -> n10;
n9 -> n11;
n10;
n10 [label="30"];
n11;
n11 [label="31"];
n9 [label="+"];
n6 [label="+"];
n3 [label="+"];
n2 [label="k"];
n1 [label="f"];
n0 [label="k"];

}