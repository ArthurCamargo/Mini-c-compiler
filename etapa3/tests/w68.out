0x561046e2d350
0x561046e2d350, 0x561046e2d510
0x561046e2d510
0x561046e2d510, 0x561046e2d580
0x561046e2d580
0x561046e2d580, 0x561046e2d670
0x561046e2d670
0x561046e2d670, 0x561046e2d5d0
0x561046e2d670, 0x561046e2d620
0x561046e2d670, 0x561046e2d760
0x561046e2d5d0
0x561046e2d5d0 [label="10"];
0x561046e2d620
0x561046e2d620 [label="11"];
0x561046e2d760
0x561046e2d760, 0x561046e2d6c0
0x561046e2d760, 0x561046e2d710
0x561046e2d760, 0x561046e2d850
0x561046e2d6c0
0x561046e2d6c0 [label="20"];
0x561046e2d710
0x561046e2d710 [label="21"];
0x561046e2d850
0x561046e2d850, 0x561046e2d7b0
0x561046e2d850, 0x561046e2d800
0x561046e2d7b0
0x561046e2d7b0 [label="30"];
0x561046e2d800
0x561046e2d800 [label="31"];
0x561046e2d850 [label="+"];
0x561046e2d760 [label="+"];
0x561046e2d670 [label="+"];
0x561046e2d580 [label="k"];
0x561046e2d510 [label="f"];
0x561046e2d350 [label="k"];
